mozilla-release/browser/themes/linux/jar.mn

```diff
@@ -51,6 +51,7 @@ browser.jar:
   skin/classic/browser/Toolbar-inverted.png
   skin/classic/browser/Toolbar-small.png
   skin/classic/browser/webRTC-indicator.css
+  skin/classic/browser/cliqz/theme.css                (cliqz/theme.css)
```

Comment:
how about putting it somewhere in `mozilla-release/browser/branding/cliqz/content` ? there is `aboutDialog.css there already